app/transformers/mdtp/OtherIndividuals.scala

```diff
@@ -16,33 +16,11 @@
 
 package transformers.mdtp
 
-import play.api.libs.json._
 import models.variation.NaturalPersonType
+import play.api.libs.json._
 
-object OtherIndividuals {
-
-  private val path = JsPath \ 'details \ 'trust \ 'entities \ 'naturalPerson
-
-  def transform(response : JsValue) : Reads[JsObject] = {
-    response.transform(path.json.pick).fold(
-      _ => {
-        JsPath.json.update(
-          path.json.put(JsArray())
-        )
-      },
-      otherIndividuals => {
-
-        val otherIndividualsUpdated = JsArray(otherIndividuals.as[List[NaturalPersonType]].map {
-          otherIndividual =>
-            Json.toJson(otherIndividual)(NaturalPersonType.writeToMaintain)
-        })
+object OtherIndividuals extends Entities[NaturalPersonType] {
 
-        JsPath.json.update(
-          path.json.prune andThen
-            path.json.put(otherIndividualsUpdated)
-        )
-      }
-    )
-  }
+  override val path: JsPath = JsPath \ 'details \ 'trust \ 'entities \ 'naturalPerson
```

Comment:
Can we use utils.Constants to minimise the repeated use of literal strings ?

Reply:
Yep good idea

Reply:
@jassalrichy have pushed that change up